packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -252,21 +287,51 @@ export function isNetwork(chainId: ChainId) {
   const isArbitrumRinkeby = chainId === ChainId.ArbitrumRinkeby
   const isArbitrumLocal = chainId === ChainId.ArbitrumLocal
 
+  const ethereumChainIds = [
```

Comment:
Have to restructure this a bit so that user-added custom chains can be associated with either an Arbitrum or an Orbit chain.